tika-parsers/src/main/resources/org/apache/tika/parser/recognition/tf/InceptionVideoRestDockerfile

```diff
@@ -60,7 +57,7 @@ RUN cmake -D CMAKE_BUILD_TYPE=RELEASE \
       -D INSTALL_C_EXAMPLES=OFF \
       -D OPENCV_EXTRA_MODULES_PATH=../../opencv_contrib-3.2.0/modules \
       -D BUILD_EXAMPLES=ON ..
-RUN make -j4 
+RUN make -j8
```

Comment:
no, this is what caused it to use too much disk/memory, so not going to change this to 8.